velox/core/PlanNode.h

```diff
@@ -1096,11 +1096,19 @@ class LocalPartitionNode : public PlanNode {
 
 class PartitionedOutputNode : public PlanNode {
  public:
+  enum class Kind {
+    kPartitioned,
+    kBroadcast,
+    kArbitrary,
+  };
+  static std::string kindString(Kind kind);
+  static Kind stringToKind(std::string kindStr);
+
   PartitionedOutputNode(
```

Comment:
This will break Prestissimo and please use backward compatibility macro to keep the old ctor.